tests/integration/test_packages.py

```diff
@@ -34,14 +35,21 @@ def test_packages(env_package, env_name, test_env, tmpdir):
     """
     env_data = utils.load_test_data(f"{env_package}.yaml")[env_name]
     client = utils.Client(test_env["api_url"], test_env["api_auth_type"], test_env.get("timeout"))
-    initial_response = client.create_new_request(
-        payload={
+
+    if env_name == "implicit_gomod":
+        payload = {
+            "repo": env_data["repo"],
+            "ref": env_data["ref"],
+        }
+    else:
+        payload = {
             "repo": env_data["repo"],
             "ref": env_data["ref"],
             "pkg_managers": env_data["pkg_managers"],
             "flags": env_data.get("flags", []),
```

Comment:
It would also be nice to change this one (in a different commit, could be a different PR.
We are always testing the request with flags. we should just add it later if it exists, like
```
if env_data.get("flags"):
    payload["flags"] = env_data.get["flags"]
```

and then adding a single test passing empty flags just to check Cachito's sanity.

Reply:
Yes, a good idea.
Or like this one: 
```suggestion
           if "flags" in env_data:
               payload["flags"] = env_data["flags"]
```

Reply:
I think we should file new issue for that and add new test to the testing doc.